validation/src/main/scala/hmda/validation/engine/ValidationError.scala

```diff
@@ -7,7 +7,8 @@ case object Quality extends ValidationErrorType
 case object Macro extends ValidationErrorType
 
 // errorID = Loan ID (LAR) or Agency Code + Respondent ID (TS)
-case class ValidationError(errorId: String, name: String, errorType: ValidationErrorType)
+case class ValidationError(errorId: String, metaData: ValidationErrorMetaData, errorType: ValidationErrorType)
+case class ValidationErrorMetaData(name: String)
```

Comment:
I don't understand yet what the `ValidationErrorMetaData` class is adding. Will it eventually hold more metadata than just the edit name?